.github/workflows/ruby.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         ruby: ["3.0", "3.1", "3.2", "3.3"]
-        appraisal: [rails_6_0, rails_6_1, rails_7_0, rails_7_1]
+        appraisal: [rails_6_0, rails_6_1, rails_7_0, rails_7_1, rails_7_2]
```

Comment:
From the test failures, it looks like rails 7.2 will not support Ruby 3.0. Can you please exclude that combination from the matrix?